test/notebook_tests/conftest.py

```diff
@@ -0,0 +1,64 @@
+import os
+
+import boto3
+import pytest
+from botocore.exceptions import ClientError
+from braket.aws.aws_session import AwsSession
+
+
+@pytest.fixture(scope="session")
+def boto_session():
+    profile_name = os.environ["AWS_PROFILE"]
+    return boto3.session.Session(profile_name=profile_name)
+
+
+@pytest.fixture(scope="session")
+def region(boto_session):
+    return boto_session.region_name
+
+
+@pytest.fixture(scope="session")
+def aws_session(boto_session):
+    return AwsSession(boto_session)
+
+
+@pytest.fixture(scope="session")
+def s3_resource(boto_session):
+    return boto_session.resource("s3")
+
+
+@pytest.fixture(scope="session")
+def s3_client(boto_session):
+    return boto_session.client("s3")
+
+
+@pytest.fixture(scope="session")
+def account_id(boto_session):
+    return boto_session.client("sts").get_caller_identity()["Account"]
+
+
+@pytest.fixture(scope="session")
+def s3_bucket(s3_resource, s3_client, account_id, boto_session):
+    """Create / get S3 bucket for tests"""
+
+    region_name = boto_session.region_name
+    bucket_name = f"amazon-braket-notebook-tests-{account_id}"
+    bucket = s3_resource.Bucket(bucket_name)
+
+    try:
+        # Determine if bucket exists
+        s3_client.head_bucket(Bucket=bucket_name)
+    except ClientError as e:
+        error_code = e.response["Error"]["Code"]
+        if error_code == "404":
+            bucket.create(
+                ACL="private",
+                CreateBucketConfiguration={"LocationConstraint": region_name},
+            )
+
+    return bucket_name
+
+
+@pytest.fixture(scope="module")
+def s3_destination_folder(s3_bucket, s3_prefix):
+    return (s3_bucket, s3_prefix)
```

Comment:
Sorry, I didn't see that AwsSession was being used above; feel free to change this back if you want.

Reply:
Yes, I was going to use this integration tests. But no issues, as it doesn't cause issues with notebook tests for now. Will add it back for integ tests.
Thanks!